### ISSUE TITLE
Walking without any buttons pressed.

### ISSUE THREAD

**Issue by AlexWHughes** [hovgaardgames/bigambitions#926]
### Build number

350

### Bug description

Upon loading up a fresh game the character starts walking/running forwards without any buttons pressed. 

### Steps to reproduce the bug

Load up new game.

**Comment by AlexWHughes** [comment]
Entering a menu stops the action. Trying full reinstall now.

**Comment by AlexWHughes** [comment]
Reinstall does not solve.  Just using a keyboard no game controllers or other hardware attached.

**Comment by jhovgaard** [comment]
Hey @AlexWHughes! Any chance you have a gamepad, steering wheel or similar connected to your computer?

**Comment by adameg2305** [comment]
Do you have a controller or any other device such as joy stick, steering wheel etc plugged into your PC when playing? @AlexWHughes

**Comment by AlexWHughes** [comment]
No, as I said in the followup comment no controllers are connected, etc.

**Comment by jhovgaard** [comment]
We only accept movement input from either WASD navigation or input devices (like mouse, gamepad, etc.). I can't think of anything that would cause this behavior.

Let's keep the issue open and see if more people can point us in the right direction.

**Comment by AlexWHughes** [comment]
Is there anything I can do to provide more diagnostic information? 

Here's a screencap with input visualization enabled via xsplit (during previous testing I have not been using xsplit

[](https://www.dropbox.com/s/b45sclfiyy9ppvz/2022-03-01_18-57-24.mp4?dl=0 )

**Comment by LPDMinecraft** [comment]
Its an error from windows or Unity I think. Its the same happening in Minecraft and Rocket League.

**Comment by AlexWHughes** [comment]
So you've experienced it as well? I don't use either of those applications. I wonder what the workaround is!

**Comment by AlexWHughes** [comment]
Might have a solve. Was able to close a process running on system and resolve. Will nail down what process it was.
